src/InteractiveWindow/Editor/InteractiveWindow.cs

```diff
@@ -555,6 +557,16 @@ private static void DoEvents()
 
         internal event Action<State> StateChanged;
 
+        internal void Undo(int count)
```

Comment:
Oops - if these methods in the "Testing" region indicated that in their name (or at least xml doc comment), so that other people don't start using them accidentally.